app/src/main/res/values-night/color_palette.xml

```diff
@@ -225,4 +225,5 @@
   <color name="color_palette_nps_selected_text_color">@color/color_def_white</color>
   <color name="color_palette_nps_selected_background_color">@color/color_def_oppia_green</color>
   <color name="color_palette_nps_unselected_background_color">@color/color_def_pale_green</color>
+  <color name="color_palette_button_shadow_color">@color/color_def_black_25</color>
```

Comment:
Is this needed if it's also in the day file? I thought that we only needed to add colors here if they were changed for night mode.

Reply:
There is really no strict guideline for this. I have looked at existing colors and there are examples of the color being included in both files depite being the same. But I feel like this approach is safer incase the color is changed in the future.